packages/react-components/src/types/Attachment.ts

```diff
@@ -44,48 +83,69 @@ export interface AttachmentMenuAction {
 
 /**
  * @beta
- * A attachment download error returned via a {@link FileDownloadHandler}.
- * This error message is used to render an error message in the UI.
  */
-export interface FileDownloadError {
-  /** The error message to display in the UI */
-  errorMessage: string;
+export interface AttachmentUploadOptions {
+  /**
+   * A string containing the comma separated list of accepted file types.
+   * Similar to the `accept` attribute of the `<input type="file" />` element.
+   * Accepts any type of file if not specified.
+   * @beta
+   */
+  acceptedMimeTypes?: string[];
+  /**
+   * Allows multiple files to be selected if set to `true`.
+   * Similar to the `multiple` attribute of the `<input type="file" />` element.
+   * @defaultValue false
+   * @beta
+   */
+  canUploadMultiple?: boolean;
```

Comment:
```suggestion
  multipleSelectionAllowed?: boolean;
```

Reply:
i feel like selection is kind of ambiguous since contoso might mistaken this with some UI related selections provided by UI library